app.js

```diff
@@ -22,4 +22,30 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => res.render('beers', { beersFromApi }))
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => res.render('random-beer', { responseFromAPI }))
+    .catch(error => console.log(error));
+});
+
+app.get('/beer-detailed', (req, res) => {
```

Comment:
This`/beer-detailed` route shouldn't be here. 
What you have bellow is what handles the `beer-detailed` hbs file that you created.